evap/evaluation/migrations/0147_unusable_password_default.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 5.1.1 on 2024-10-07 20:09
+
+import django.contrib.auth.hashers
+import functools
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("evaluation", "0146_grade_reminder_template"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="userprofile",
+            name="password",
+            field=models.CharField(
+                default=functools.partial(django.contrib.auth.hashers.make_password, *(None,)),
```

Comment:
this is necessary so that django does not create a migration.... I get this otherwise:
`You have 1 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): evaluation.`